storm-core/test/clj/backtype/storm/nimbus_test.clj

```diff
@@ -846,16 +846,6 @@
                   NIMBUS-SLOTS-PER-TOPOLOGY 8}]
     (letlocals
       (bind topology (thrift/mk-topology
-                        {"1" (thrift/mk-spout-spec (TestPlannerSpout. true) :parallelism-hint 0 :conf {TOPOLOGY-TASKS 1})}
```

Comment:
@mjsax 
thrift/mk-spout-spec throws IllegalArgumentException since it actually uses TopologyBuilder.
Removing this test makes sense since we cannot even try to submit topology with dop 0 now.